x/gov/common_test.go

```diff
@@ -0,0 +1,97 @@
+package gov_test
+
```

Comment:
probably not needed?

Reply:
We need `SortAddresses` because the addresses returned by `getQueriedDeposits` are in sorted order.Random tests have the potential to fail